src/notation/internal/notationuiactions.cpp

```diff
@@ -1181,6 +1181,18 @@ void NotationUiActions::init()
             alist.push_back(a.code);
         }
         m_actionCheckedChanged.send(alist);
+
+        m_controller->currentNotationInteraction()->scoreConfigChanged().onReceive(this, [this](ScoreConfigType configType) {
+            std::unordered_map<ScoreConfigType, std::string> configActions = {
```

Comment:
Better to mark this collection as static const:

static const std::unordered_map<ScoreConfigType, std::string> configActions = {
...

Reply:
It can't compile with ```static const``` so I left it with just ```static```.

Reply:
If you use [`at()`](https://en.cppreference.com/w/cpp/container/unordered_map/at) instead of [`operator []`](https://en.cppreference.com/w/cpp/container/unordered_map/operator_at), you can make `configActions` const. 

Because [] performs an insertion if the key is not found, that one is not const; at() throws an exception when the key is not found. But as long as all ScoreConfigType enum values are in the map, that won't be a problem. 

@RomanPudashkin Do you think it would be good to use at()?

Reply:
Sure, I can change it to ```at()```.